modules/social_features/social_profile/social_profile.links.menu.yml

```diff
@@ -1,5 +1,5 @@
 social_profile.settings:
- title: 'Profile privacy settings'
- description: 'Privacy settings for Social Profile module.'
+ title: 'Profile settings'
+ description: 'Profile settings for Social Profile module.'
```

Comment:
@dmytriikaiun I see the point of changing the Title since it now not only privacy settings.
But I think we can keep old description since `Profile settings for Social Profile module` is a bit clumsy as well ;)